doc/guides/contributing/code-of-conduct.md

```diff
@@ -1,4 +1,4 @@
-# Code of Conduct
+# Code of conduct
```

Comment:
I'm not sure we want to do this, the CoC is referenced in the line below with two uppercase `C`s and in the CoC document itself.

Reply:
So, do I remove this change or use sentence case in the CoC reference in line below?

Reply:
I think it should follow https://docs.microsoft.com/en-us/style-guide/capitalization#title-style-capitalization, so keep `Code of Conduct`.